indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -286,6 +286,45 @@ $dropdown-transition: $dropdown-transition-step ease-out;
     }
 }
 
+.occurrences-widget {
+    @extend .datetime-widget;
+
+    .occurrence + .occurrence {
+        margin-top: 5px;
+    }
+
+    .timepicker {
+        width: 26% !important;
+    }
+
+    .durationpicker {
+        width: 20% !important;
+
+        // the spinner looks awful in the field, let's just hide it
```

Comment:
Even in Chrome? It should be just arrow-up and arrow-down. Does it look that awful?